docs/tutorial/additional-tutorials.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Tutoriales adicionales
+título: Tutoriales adicionales
```

Comment:
`title`  is variable's name from markdown frontmatter. It shouldn't be translated because it's metadata to build pages.